data/jeesus-bock.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Jeesus Bock",
+  "bio": "Alternative lifestyles, over 40y/o. Vegan. Software developer with VueJS and Go. Knowledge of many more languages and frameworks.",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/JeesusBock",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Very freeform OSS project, open to newbies too.",
+      "url": "https://github.com/jeesus-bock/animalfarm/",
```

Please add a valid icon for this link. 

```suggestion
      "url": "https://github.com/jeesus-bock/animalfarm/",
      "icon": "FaGithub"
```